katunold/ekalaamu
# ft-add-password-reset-component

#### What does this PR do?
- generate password reset component
- design UI for the resetting password
#### Description of Task to be completed?
- design dialog/model for users to reset password
- create a service and component to handle password reset functionality
- write tests for the feature

#### How should this be manually tested?
- clone this repository, start the server and on the login page, click the password reset link.
- A dialog box should pop. Input the registered email to receive a password reset link

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?


#### Postman Documentation


#### Screenshots (If applicable)
